3rdparty/btfhub.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
I'm installing it in our "tracee-make" container (but not making it the default). The reason for me to install is to give a better experience for "tracee-make" container shell users (set -o vi mode is more complete, for example, among others). 

For the script, I'm using "bash"-isms on it. I can try to use /bin/sh by default, but, since we're installing it I thought it would be okay to for the script as well.